### ISSUE TITLE
When I install "@testing-library/react": "^11.2.5" in an existing project with tslint, I get missing kcd-scripts warning

### ISSUE THREAD

**Issue by justin-calleja** [testing-library/react-testing-library#883]
- `@testing-library/react`: 11.2.5
- Testing Framework and version: jest at 26.6.3
  
- DOM Environment: jsdom at 11.12.0
  

### What you did:

install "@testing-library/react": "^11.2.5" via adding directly in package.json and `lerna bootstrap`.

### What happened:

> Cannot read file '.../node_modules/@testing-library/react/node_modules/kcd-scripts/shared-tsconfig.json'.

### Problem description:

A dependency of react-testing-library is not being installed. In react-testing-library's package.json:

```json
 "eslintConfig": {
    "extends": "./node_modules/kcd-scripts/eslint.js",
```

So my editor now requires the existance of that eslint.js file.

### Suggested solution:

Add the missing kcd-scripts in dependencies of react-testing-library.

When I install `kcd-scripts` alongside `@testing-library/react` in my package devDependencies (and restart TS server in vs code), then the error goes away. But I would expect that the expected behaviour is to just install `@testing-library/react` and not get the error in the first place 🙂

**Comment by MatanBobi** [comment]
Hi @justin-calleja, thanks for taking the time to open this :)
It looks like you're looking for the `shared-tsconfig.json` which is being used in our [tsconfig](https://github.com/testing-library/react-testing-library/blob/master/types/tsconfig.json).
The eslint configs you've added here are there for a few years now and nothing was changed there. Also, the eslint+tsconfig are only used as a dev dependency..
Can you please add your tsconfig so we'll be able to look into what's going on and why you're trying to load the `shared-tsconfig.json` file?

**Comment by ryan-rushton** [comment]
Hi I am seeing something similar

>Cannot read file '/Users/ryanrushton/dev/evdb-frontend/node_modules/@testing-library/react/node_modules/kcd-scripts/shared-tsconfig.json'.

This is what the @testing-library package in my node_modules looks like

![Screen Shot 2021-03-30 at 2 55 48 pm](https://user-images.githubusercontent.com/7344652/112931567-221ec400-9168-11eb-9b27-a8c7798e36f7.png)

And this is my tsconfig.json

```json
{
  "compilerOptions": {
    "target": "es5",
    "lib": [
      "dom",
      "dom.iterable",
      "esnext"
    ],
    "allowJs": true,
    "skipLibCheck": true,
    "esModuleInterop": true,
    "allowSyntheticDefaultImports": true,
    "strict": true,
    "forceConsistentCasingInFileNames": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "noEmit": true,
    "jsx": "react-jsx",
    "baseUrl": "src",
    "noFallthroughCasesInSwitch": true
  },
  "include": [
    "src"
  ]
}

```

**Comment by MatanBobi** [comment]
Thanks @ryan-rushton, This Issue is fixed in version 11.2.6 :tada:

The release is available on:
- `npm package (@latest dist-tag)`
- [GitHub release](https://github.com/testing-library/react-testing-library/releases/tag/v11.2.6)
